data_steward/bq_utils.py

```diff
@@ -761,25 +786,11 @@ def load_table_from_csv(project_id, dataset_id, table_name, csv_path=None, field
         fields = json.load(open(fields_filename, 'r'))
     field_names = ', '.join([field['name'] for field in fields])
 
-    # template for formatted values.
-    # string, dates and timestamps can be passed as string
-    # integer, float and boolean need to be passed without string quotes
-    insert_format_vals = '({cols})'.format(cols=', '.join(['"{' + field['name'] + '}"' if field["type"] == 'string'
-                                                           or field["type"] == 'date' or field["type"] == 'timestamp'
-                                                           else '{' + field['name'] + '}' for field in fields]))
-
-    pair_exprs = []
+    row_exprs = []
     for mapping_dict in table_list:
-        converted_dict = dict()
-        for k, v in mapping_dict.items():
-            for field in fields:
-                if field['name'] == k and field['type'] == 'integer':
-                    converted_dict[k] = int(v)
-                else:
-                    converted_dict[k] = v
-        pair_expr = insert_format_vals.format(**converted_dict)
-        pair_exprs.append(pair_expr)
-    formatted_mapping_list = ', '.join(pair_exprs)
+        pair_expr = get_sql_row_expr(mapping_dict, fields)
```

Comment:
opt: maybe a list comprehension here `row_exprs = [get_sql_row_expr(t, fields) for t in table_list]`

Reply:
yes- I typically only use list comprehensions when they are as simple as this. will do.